app/controllers/tags_controller.rb

```diff
@@ -0,0 +1,38 @@
+class TagsController < ApplicationController
+  before_action :authenticate_user!
+
+  def new
+    @tag = Tag.new
+    game_id = find_game_id
+    @game = Game.find(game_id)
+  end
+
+  def create
+    game_id = find_game_id
+    @game = Game.find(game_id)
+    name = tag_params[:name].downcase
+    tag = Tag.new(name: name)
```

Comment:
You can move this logic to the model and use a `before_validate` callback to downcase the name before it is saved.